### ISSUE TITLE
Add support for returning non random abstract types

### ISSUE THREAD

**Issue by milansmolik-dragon** [APIs-guru/graphql-faker#73]
I have had a very specific problem: I have an app, which uses interface `Card` extended by both `Debit Card` and `CreditCard`. They are used by `DebitAccount` and `CreditAccount` respectively. However when I receive faked `CreditCard` type for `DebitAccount`, the app is not built to handle that. 

It would be best for me if I could return `DebitCard` for `DebitAccount` only, and not to be limited by a random selection from all `Card` types.

Would there be interest in such functionality? I have already implemented it in my fork, so I can open a PR for it whenever.

**Comment by IvanGoncharov** [comment]
@milansmolik-dragon I added support for `__typename` in `@examples` values 72f59cac5cca0fad0597ad65bf15e34569fdb32e
and released it as `2.0.0-rc.2` on NPM.
It allows you to do the following:
```graphql
type Query {
  card: Card @examples(values:[{ __typename: DebitCard }])
}

interface Card {
  id: ID!
}

type DebitCard implements Card {
  id: ID!
}

type CreditCard implements Card {
  id: ID!
  creditLimit: Float
}
```
Can you please test it?
Does it solve your issue?

**Comment by milansmolik-dragon** [comment]
Oh yeah, I used an older build for that. This one works great, thanks!
